os/net/net_initialize_netmgr.c

```diff
@@ -104,6 +107,9 @@ void net_initialize(void)
 {
 #ifdef CONFIG_VIRTUAL_WLAN
 	vwifi_start();
+#endif
+#ifdef CONFIG_VIRTUAL_BLE
+    vble_kernel_thread();
```

Comment:
Not here, thread should be executed when driver initialized. also thread should be destroyed when driver deinitialized.

Reply:
Virtual BLE gets executed from this point after registration. Naming convention creates confusion. Made necessary changes.

Reply:
Network should not coupled with ble driver, this is not good structure, imo.

Reply:
> Not here, thread should be executed when driver initialized. also thread should be destroyed when driver deinitialized.

This is driver initialization point. We will check for thread deinitialization.